addons/wowl/static/src/components/navbar/navbar.ts

```diff
@@ -16,14 +16,59 @@ export class NavBar extends Component<{}, OdooEnv> {
   actionManager = useService("action_manager");
   menuRepo = useService("menus");
 
+  private _debouncedAdapt = utils.debounce(this._adapt.bind(this), 250);
+
   constructor(...args: any[]) {
     super(...args);
     hooks.onMounted(() => {
-      this.env.bus.on("MENUS:APP-CHANGED", this, () => this.render());
+      this.env.bus.on("MENUS:APP-CHANGED", this, () => {
+        this._debouncedAdapt();
```

Comment:
maybe you should call the non debounced version here, no?

Reply:
Oh yeah ! I totally should. And also, remove the _render() call here, as _adapt() will call _render() itself.